config/sync/metatag.metatag_defaults.front.yml

```diff
@@ -13,7 +13,6 @@ tags:
   icbm: '38.044810, -84.495460'
   canonical_url: '[site:url]'
   geo_placename: 'Lexington Kentucky'
-  news_keywords: 'City of Lexington KY'
```

Comment:
Just curious why this was removed.

Reply:
Looks like google dropped it.

https://www.drupal.org/project/metatag/issues/2973351

Reply:
Word.